client/settings/wcpay-upe-toggle/context.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * External dependencies
+ */
+import { createContext } from 'react';
+
+const WcPayUpeContext = createContext( {
+	isUpeEnabled: false,
+	setIsUpeEnabled: () => null,
```

Comment:
I can't think of any examples but doesn't doing this increase the risk of not defining the function and not catching the fact that we didn't define it? In other words, isn't it safer to provide no defaults so the code yells "no key `setIsUpeEnabled`!" at us when it's not defined?

Reply:
I like to provide some sensible defaults, maybe I'm accustomed to using TS, where if there's a tiny chance something is not defined, you need to check its existance, even if you _know_ it's there :D 

Let me know if I misunderstood, but if somebody defined a context value and didn't define `setIsUpeEnabled`, the code _would_ indeed yell that the method is not defined.

Reply:
Yeah, this is what I'd consider desired! 😄 You either define it correctly or get yelled at which would reduce the risk of forgetting to define `setIsUpeEnabled()` and then wondering "why isn't this working?"

No strong feelings about this either but I'll be happy to learn something new if your approach is better!